### ISSUE TITLE
[Windows/Linux] Desktop icon badge should display number of unread notifications/messages

### ISSUE THREAD

**Issue by morebrownies** [OpenBazaar/OpenBazaar-Client#372]
<img width="354" alt="screenshot 2015-12-12 11 17 31" src="https://cloud.githubusercontent.com/assets/13066886/11763072/0dccaf5e-a0c2-11e5-9c4e-920ccf84983e.png">

**Comment by morebrownies** [comment]
@jjeffryes @rmisio If either of you have extra time, this could be a good one to try to squeeze in. If I walk away from my computer for a few moments and a notification comes in while I'm away, I won't have any indicator to know to check OpenBazaar. If we can't get it in before Tuesday, we should definitely try to get it in before the main net launch.

**Comment by hoffmabc** [comment]
I don't think this is a huge lift if we can do it. I'll check to see if there's anything os specific about it.

**Comment by jjeffryes** [comment]
This has been done on OSX.

For Windows and Linux, the plan is to use something like this https://gist.github.com/johnthedebs/c22a04fcd08e598e69b8

**Comment by mariodian** [comment]
Everytime I get a new message for orders, I don't see any notification about it. I hear the notification sound and can read the new message in order details, but can't see the notification in the notification menu. I can't see notification badges too. OSX
